oaknational/Oak-Web-Application
# refactor: change drop down select role to listbox - LESQ-899

## Description

# Acceptance Criteria:

- [ ]  refactor the newsletter sign up form/contact us form so that the role selector is using the `<select>` element with an accessible `<label>`
    - There is some context provided in the audit ticket linked above
    - They recommend using a `combobox` if we can’t use `select` -  with use of hotkeys listbox seems like the corretc option.



Music year: {owa_music_year}

- Changes role of drop down select to 'combobox' from 'button' , initially tried 'select' but this would not work.  AC recommended to use combobox.   `aria-labelledby` is used to link to the rotated input label


![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/aae876f8-d38d-4e17-a2e5-0963615c5043)

## How to test

1. Go to https://deploy-preview-2537--oak-web-application.netlify.thenational.academy
Screen reader on newsletter should read combobox rather than button.

-Contact  - /contact-us
-Home page -  / 




## Checklist
